### PR TITLE
Lean slice

### PR DESCRIPTION
**Pull request by lpil**
### Motivation

We want everyone to see how lean we are.

### Test plan

Go to the what-we-do page, check there's a lean slice there.

### Pre-merge checklist

- [x] Reviews
  - [x] Code review
  - [x] Design review
- [ ] Manual testing
  - [x] **Ubuntu Linux 16.04 Chrome**
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
- [ ] Tester approved

### CODE REVIEW

**Comment by AndrewBestbier** [comment]
Move the tech slice below the lean slice

**Comment by lpil** [comment]
It is already :)

File: site/pages/what-we-do/index.js [hunk 1]
```diff
@@ -86,3 +76,22 @@ export default function whatWeDo() {
     </div>
   );
 }
+
+export default function whatWeDo() {
+  return (
+    <div>
+      <div className={styles.pageHeaderContainer}>
+        <h1 className={styles.pageHeader}>
```
Comment (asavin) [inline site/pages/what-we-do/index.js]: Can we combine these 2 h1:s into one, with spans? Otherwise screen reader splits them and it's become less readable.

File: site/pages/what-we-do/lean-slice/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      </h2>
+
+      <div className={styles.content}>
+        <div className={styles.leanImageWrapper} >
+          <img role="presentation" className={styles.leanImage} src={leanImg} />
+        </div>
+
+        <div className={styles.pointsWrapper}>
+          <p className={styles.point}>
+            <span className={styles.bold}>Visualise the workflow</span> on a
```
Comment (asavin) [inline site/pages/what-we-do/lean-slice/index.js]: would `<strong>` be more semantically correct?

File: site/pages/what-we-do/index.js (hunk 1 above)
Comment (lpil) [inline site/pages/what-we-do/index.js]: It makes more sense as it is to me.

File: site/pages/what-we-do/lean-slice/index.js (hunk 2 above)
Comment (lpil) [inline site/pages/what-we-do/lean-slice/index.js]: :+1:

File: site/pages/what-we-do/index.js (hunk 1 above)
Comment (lpil) [inline site/pages/what-we-do/index.js]: Let's do this in the ally PR rather than this one as this one is unrelated to the what we do header.

Comment (zoeabryant) [inline site/pages/what-we-do/index.js]: Done
```
<h1 className={styles.pageHeader}>
          Do the <span className={styles.redText}>right</span> thing.<br />
          Do the thing <span className={styles.redText}>right</span>.
</h1>
```
